EndlessClient/Dialogs/GuildDialog.cs

```diff
@@ -31,6 +39,8 @@ private enum GuildDialogState
         private readonly IContentProvider _contentProvider;
         private readonly ICurrentMapStateProvider _currentMapStateProvider;
         private readonly IENFFileProvider _enfFileProvider;
```

Comment:
I don't see where `_enfFileProvider` is being used - can you do a quick check and remove any dependencies that aren't in use in the dialog? They can always be added as needed later

Reply:
Removed unused dependencies